src/GitHub.Exports/Exports/ExportForProcess.cs

```diff
@@ -0,0 +1,33 @@
+﻿using System;
+using System.Diagnostics;
+using System.ComponentModel.Composition;
+
+namespace GitHub.Exports
+{
+    /// <summary>
+    /// Only expose export when executing in specific named process.
+    /// </summary>
```

Comment:
Probably worth adding a `<remarks>` section here explaining why this is needed and linking to the relevant issue.